### PR TITLE
fix statistics calculation for instanced geometry

### PR DESCRIPTION
**Pull request by BAntDit**
Hi, 
now instanced geometry does not increase number of calls, verticies and faces - there are some changes to fix that issue.

### CODE REVIEW

**Comment by WestLangley** [comment]
Yay! (not tested, though.)

Making sure `renderer.info` is tallied correctly is important.

File: src/renderers/webgl/WebGLBufferRenderer.js [hunk 1]
```diff
@@ -37,11 +37,23 @@ THREE.WebGLBufferRenderer = function ( _gl, extensions, _infoRender ) {
 
 		if ( position instanceof THREE.InterleavedBufferAttribute ) {
 
-			extension.drawArraysInstancedANGLE( mode, 0, position.data.count, geometry.maxInstancedCount );
+			var count = position.data.count;
+
+			extension.drawArraysInstancedANGLE( mode, 0, count, geometry.maxInstancedCount );
+
+			_infoRender.calls ++;
```
Comment (mrdoob) [inline src/renderers/webgl/WebGLBufferRenderer.js]: We can probably put this one out of the conditional 😇

**Comment by BAntDit** [comment]
put statistics calculation out of the conditional

**Comment by mrdoob** [comment]
Lovely 😊

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**